File: problems/0004_median-of-two-sorted-arrays/concat-sort/Solution.cs
public class Solution {
    public double FindMedianSortedArrays(int[] nums1, int[] nums2) {
        var nums = new int[nums1.Length + nums2.Length];
        Array.Copy(nums1, 0, nums, 0, nums1.Length);
        Array.Copy(nums2, 0, nums, nums1.Length, nums2.Length);
        Array.Sort(nums);
        var i = nums.Length / 2;
        return (nums.Length % 2 == 0 ? (nums[i - 1] + nums[i]) / 2.0 : nums[i]);
    }
}

File: problems/0004_median-of-two-sorted-arrays/concat-sort/Solution.cpp
using namespace std;

class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        vector<int> nums(nums1);
        copy(nums2.begin(), nums2.end(), back_inserter(nums));
        sort(nums.begin(), nums.end());
        const int size = nums.size(),
                  i = size / 2;
        return (size % 2 == 0 ? (nums[i - 1] + nums[i]) / 2.0 : nums[i]);
    }
};
